src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageService`IVsLangaugeInfo.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.TextManager.Interop;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+    [Guid(RazorLSPConstants.LanguageServiceString)]
+    internal partial class RazorLanguageService : IVsLanguageInfo
```

Comment:
Is the ` intentional in the file name to denote the partial class?

RazorLanguageService`IVsLangaugeInfo.cs

Reply:
Yup 😄